recipes-tools/fluent-bit/files/fluent-bit.init

```diff
@@ -11,7 +11,7 @@
 
 NAME=td-agent-bit
 DAEMON=/usr/bin/$NAME
-CONFIG_FILE=/etc/td-agent-bit/td-agent-bit.conf
+CONFIG_FILE=/etc/fluent-bit/fluent-bit.yaml
```

Comment:
I originally had them as fluent-bit everywhere but the recipe installs the binary with the name td-agent-bit. 
We can rename everything, including making the recipe installs the binary with a different name or keep the original naming of the binary and make it consistent with the rest